packages/mail/src/classes/mail-service.js

```diff
@@ -184,8 +197,11 @@ class MailService {
         data.substitutionWrappers = this.substitutionWrappers;
       }
 
+      //Deep merge default data
+      const dataWithDefaults = deepmerge(this.defaultData, data);
```

Comment:
Tests needed to ensure the default data is actually getting merged.

Reply:
Need some help on testing, documentation is not very clear to me https://github.com/sendgrid/sendgrid-nodejs/blob/main/CONTRIBUTING.md#testing
Seems there is no prism:install and prism command. Is there a reason why this is not on devDependencies?
Thanks

Reply:
Testing section updated with latest instructions.